npc/re/mobs/dungeons/moro_vol.txt

```diff
@@ -0,0 +1,23 @@
+//===== rAthena Script =======================================
+//= Flame Basin Monster Spawn Script
+//===== Description: =========================================
+//= Spawns monsters on Flame Basin.
+//===== Additional Comments: =================================
+//= 1.0 First version. [Capuche]
+//============================================================
+
+moro_vol	monster	Angry Mimic	3061,5
+moro_vol	monster	Fire Pit	3026,10
+moro_vol	monster	Sonia	3028,30
+moro_vol	monster	Incarnation of Morocc	1918,15
+moro_vol	monster	Incarnation of Morocc	1919,15
+moro_vol	monster	Incarnation of Morocc	1920,15
+moro_vol	monster	Incarnation of Morocc	1921,15
+moro_vol	monster	Fire Condor	3020,30
+moro_vol	monster	Fire Sandman	3021,30
+moro_vol	monster	Fire Frilldora	3022,30
+moro_vol	monster	Fire Golem	3023,30
+moro_vol	monster	Fire Bug	3027,10
+moro_vol	monster	Blazing Morocc Reincarnation	3039,1
+moro_vol	monster	Distorted Morocc Reincarnation	3040,1
+moro_vol	monster	Freezing Morocc Reincarnation	3041,1
```

Comment:
i think this are wrong definition

it should be something like this

moro_vol,0,0	monster	Fire Pit	3026,10,5000

Reply:
In commit https://github.com/rathena/rathena/pull/3002/commits/0d276f864da9030c3a9969a07ea64b6f51ae5809 `Adjusted permanent monster spawn to 0 (random spawn) xy by default` : the map name alone define a random spawn.
The respawn timer is optionnal. Monster never respawn faster than 5 seconds : https://github.com/rathena/rathena/blob/master/src/map/mob.cpp#L1082